leancloud/file_.py

```diff
@@ -133,7 +133,7 @@ def save(self):
             base64.encode(self._source, output)
             self._source.seek(0)
             output.seek(0)
-            hex_octet = lambda: hex(int(1 + random.random() * 0x10000))[2:]
+            hex_octet = lambda: hex(int(1 + random.random()) * 0x10000)[2:]
```

Comment:
In [3]: os.urandom(1).encode('hex')
Out[3]: '6a'

In [4]: os.urandom(1).encode('hex')
Out[4]: '68'

In [5]: os.urandom(1).encode('hex')
Out[5]: 'b2'

In [6]: os.urandom(1).encode('hex')
Out[6]: '50'